lang/cs/Org.Apache.REEF.IMRU/API/IIMRUClient.cs

```diff
@@ -32,5 +33,10 @@ public interface IIMRUClient
         /// <param name="jobDefinition">IMRU job definition</param>
         /// <returns>Result of IMRU</returns>
         IEnumerable<TResult> Submit<TMapInput, TMapOutput, TResult>(IMRUJobDefinition jobDefinition);
+
+        /// <summary>
+        /// DriverHttpEndPoint returned by IReefClient after job submission
+        /// </summary>
+        IDriverHttpEndpoint DriverHttpEndpoint { get; }
```

Comment:
Please mark this with `[Unstable]` as it relies on an API which itself is `[Unstable]`.